### DISCUSSION TITLE
alignment3

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#23] [Announcements]
# alignment3

Exploring the alignment of individual and collective interests in the realm of politics and society.

https://nonzerosum.games/alignment3.html

**Comment by andrewtaneglen** [comment]
Great post!

Aristotle's take on self-sufficiency is really interesting, at least compared to what we think, prima-facie, when someone mentions the importance of self-sufficiency in modern parlance. Distinct from the 'rugged individualist' living off the land, eating their own kill, living in a self-made log cabin, Aristotle's self-sufficient dude is more like a 'contributing member of society' - who recognises the vast non-zero-sum value of a society, contributes to this, and benefits greatly from it. In this sense it should be the legislator's aim to maximuse the production of self-sufficient citizens - of not the apocalyptic fantasist (and suspiciously fascistic) kind.

Another interesting example of self-interest overcoming perceived self-interest, is Adam Smith's argument that it would actually cost less (or similar) and generate more productivity to free slaves and pay them according to their output - the free-men being more motivated to work harder (for more pay), and more efficient in their self-maintenance than the former slave owners - resulting in greatly improved economic productivity. The problem for slave-owners was that many lacked the capacity to operate their farms in a free-market - likely to be quickly outcompeted by freed slaves who actually knew what they were doing. So they held on to their farming monopolies, even as their production fell away, ultimately impoverishing themselves and their local economies for future generations.

**Comment by andrewtaneglen** [comment]
As corny as it might sound, Arnold Schwarzenegger has a great speech about the myth of the 'self-made man':
https://www.youtube.com/watch?v=RJsvR_gSEjg&t=270s

**Reply by jamesstephenbrown** [reply]
Man I love Arnie, he may not have been a self-made man, but for sure that is a man who has "self-actualised".

**Comment by MattBallAZ** [comment]
Nixon also proposed a great health care system, which Dems turned down because they thought that they could get more. DOH!
